src/scheduler.js

```diff
@@ -162,8 +162,14 @@ class Scheduler extends events.EventEmitter {
       errJson = `(error formatting JSON: ${e})`;
     }
 
-    this.notify.email({
-      address: email,
+    let email = this.createEmail(hook, err, errJson);
+    this.notify.email(email);
+    this.notify.lastEmail = _.merge({}, email, {err, errJson});
```

Comment:
Something like this would do:

```
notify = {
  ...
  email: (payload) => {
    ...
    notify.lastEmail = payload
  }
};

module.exports = notify;
```